lib/from_erlang.ml

```diff
@@ -4,13 +4,15 @@ open Obeam
 
 module F = Abstract_format
 
-let unit : expr = Val (Int 0)
+let unit : expr = Constant (Number 0)
 
 let const_of_literal = function
   | F.LitAtom (_line_t, name) -> Atom name
-  | LitInteger (_line_t, i) -> Int i
-  | LitBigInt (_line_t, z) -> BigInt z
-  | LitString (_line_t, s) -> String s
+  | LitInteger (_line_t, i) -> Number i
+  | LitBigInt (_line_t, z) ->
+     raise Known_error.(FialyzerError (NotImplemented (make_issue ~url:"TODO: bigint literal")))
+  | LitString (_line_t, s) ->
+     raise Known_error.(FialyzerError (NotImplemented (make_issue ~url:"TODO: string literal")))
```

Comment:
I think string literal is not rarely appeared in Erlang statements (not types) but I'm OK it is TODO for the first milestone.